pallets/automation-time/src/benchmarking.rs

```diff
@@ -76,6 +76,26 @@ benchmarks! {
 		let task_id: T::Hash = schedule_tasks::<T>(caller.clone(), time, 1);
 	}: schedule_notify_task(RawOrigin::Signed(caller), vec![10], time, vec![4, 5])
 
+	// First transfer task for a slot
+	schedule_transfer_task_new_slot {
+		let caller: T::AccountId = whitelisted_caller();
+		let recipient: T::AccountId = whitelisted_caller();
+		let time: u64 = 120;
+		let count: u8 = 1;
+
+		let task_id: T::Hash = schedule_tasks::<T>(caller.clone(), time, 1);
```

Comment:
since this is for a new slot you shouldn't schedule. Also since we know the existing slot takes longer we really only need 1 benchmark for the existing. I need to clean up this file to just one have per extrinsic (the bigger one)